.bazelci/build_bazel_binaries.yml

```diff
@@ -49,3 +49,13 @@ platforms:
     build_targets:
       - "//src:bazel.exe"
       - "//src:bazel_nojdk.exe"
+  windows_arm64:
+    build_flags:
+      - "--copt=-w"
+      - "--host_copt=-w"
+      - "-c"
+      - "opt"
+      - "--cpu=x64_arm64_windows"
```

Comment:
Shouldn't it just be `--cpu=arm64_windows`?

Reply:
Yes, I had it as `arm64_windows` but changed it thinking there might be bootstrapping issue. At least for the first ARM64 `Bazel` build, we will be using `Bazel` which doesn't recognize `arm64_windows` as CPU.

Reply:
Oh, I see, you are right.

Reply:
Thanks, I removed some unnecessary build flags.

Reply:
Adding the support for `windows_arm64` here: https://github.com/bazelbuild/continuous-integration/pull/1320